vividus-studio-server/vividus-studio-plugin/src/test/java/org/vividus/studio/plugin/testdata/TestData.java

```diff
@@ -0,0 +1,60 @@
+/*-
+ * *
+ * *
+ * Copyright (C) 2020 - 2022 the original author or authors.
+ * *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * *
+ */
+
+package org.vividus.studio.plugin.testdata;
+
+import java.util.List;
+
+import org.vividus.studio.plugin.model.Parameter;
+import org.vividus.studio.plugin.model.StepDefinition;
+
+public class TestData
```

Comment:
personally I'm not a fan of such approach, it leads to maintenance issues, when test data should be updated for one class with unit tests and should be preserved for another one